### DISCUSSION TITLE
Error on installing

### DISCUSSION THREAD

**Discussion by fokuleh** [mikulash/Octoprint-detector2#3] [Q&A] [answered]
I try install the plugin but got error, now i try unistall to reinstall again but got the following error:

Uninstalling plugin "Detector2 Plugin"/home/pi/oprint/bin/python3 -m pip --disable-pip-version-check uninstall --yes -ctoPrint-Detector2Usage:/home/pi/oprint/bin/python3 -m pip uninstall [options] <package> .../home/pi/oprint/bin/python3 -m pip uninstall [options] -r <requirements file> ...no such option: -cDone!Done!

**Comment by mikulash** [comment]
Sorry to hear that. I'm gonna try to replicate the error and hopefully will be able to fix it

**Comment by mikulash** [comment, marked as answer]
Hi, created a fix. 
Until it gets updated in a plugin repository, you can install it from a file that you can get here https://github.com/mikulash/Octoprint-detector2/releases/tag/0.1.1
